Frameworks/CoreFoundationAdditions/CFStringHelpers.mm

```diff
@@ -0,0 +1,42 @@
+//******************************************************************************
+//
+// Copyright (c) Microsoft. All rights reserved.
+//
+// This code is licensed under the MIT License (MIT).
+//
+// THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+// IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+// FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+// AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+// LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+// OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
+// THE SOFTWARE.
+//
+//******************************************************************************
+
+#import <CoreFoundation/CFString.h>
+#import <Starboard/SmartTypes.h>
+#import <CFInternal.h>
+#import "CFFoundationInternal.h"
+
+CFStringRef _CFStringCreateReversedString(CFStringRef string) {
```

Comment:
Lets not do this.  the _reverseString was added because some other code was using it.  Given that there is only one instance where _reverseString was used, there is no need to go overboard here.  

---
In reply to: [110781327](https://github.com/Microsoft/WinObjC/pull/2479#discussion_r110781327) [](ancestors = 110781327)